lib/helper.js

```diff
@@ -94,6 +94,32 @@ function stubComponent(tag, children, showDataProps) {
 }
 
 /**
+ * Wraps component in a context creating component
+ *
+ * @method withContext
+ * @param {ReactComponent} Component class
+ * @return {ReactElement} react class with context setup
+ **/
+function withContext(context, Component) {
```

Comment:
on all other methods that take a component or element it is passed as the first arg I think we should do the same here:

```
function withContext(Component, context) { ... }
```